tests/model/issue.model.spec.ts

```diff
@@ -43,3 +48,125 @@ describe('Issue model class', () => {
         });
     });
 });
+
+describe('Issue', () => {
+    let newIssueDispute: IssueDispute;
+    let newTesterResponse: TesterResponse;
+
+    const noReportedDescriptionString = 'No details provided by bug reporter.\n';
+    const tutorResponseStringHeader = '# Tutor Moderation\n\n';
+
+    beforeEach(() => {
+        newIssueDispute = new IssueDispute('Cannot Work', 'Help Please');
+        newTesterResponse = new TesterResponse('Cannot Work', 'Help Please', '- [ ] Not Done', 'Reason');
+    });
+
+    it('.clone() should intialise the cloned issue with the correct phase and team', () => {
+        const dummyTeam = new Team({
```

Comment:
I actually meant that we can define `dummyTeam` / `dummyIssue` / `dummyIssueWithTeam` / etc once at the top level of this `describe` block.

i.e.
```
describe('Issue', () => {
  const dummyTeam = new Team({ ... });
  const dummyIssue = ..
  ...
  const newIssueDispute = new IssueDispute('Cannot Work', 'Help Please');

   // tests here
```

Reply:
Ahhh I see... my bad ><